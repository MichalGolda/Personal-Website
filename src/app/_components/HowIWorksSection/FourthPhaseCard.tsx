import PhaseCard from "./PhaseCard";
import { PhaseCardColors } from "./constants";

const FourthPhaseCard = () => (
  <PhaseCard
    className="w-full md:w-1/2 md:ml-[calc((50%/3)*3)]"
    headline="faza 4"
    title="Wdrożenie"
    color={PhaseCardColors.PHASE_FOURTH}
    body="Po zatwierdzeniu projektu przechodzę do wdrożenia na serwer produkcyjny. Dodatkowo integruję narzędzia takie jak Google Analytics, Sentry, Google Search Console. Przeprowadzam również kompleksową optymalizację SEO."
    data-aos="fade-right"
    data-aos-duration="2000"
  />
);

export default FourthPhaseCard;
