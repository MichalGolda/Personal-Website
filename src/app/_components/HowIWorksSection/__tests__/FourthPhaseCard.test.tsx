import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import FourthPhaseCard from "../FourthPhaseCard";

describe("FourthPhaseCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<FourthPhaseCard />);

    expect(container).toMatchSnapshot();
  });
});
