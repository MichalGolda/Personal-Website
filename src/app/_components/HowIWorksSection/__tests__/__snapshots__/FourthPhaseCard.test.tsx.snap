// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourthPhaseCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 border border-solid border-lightGrey p-4 bg-white w-full md:w-1/2 md:ml-[calc((50%/3)*3)]"
    data-aos="fade-right"
    data-aos-duration="2000"
  >
    <div
      class="flex flex-col"
    >
      <span
        class="before:block flex items-center gap-x-2 text-sm md:text-base font-semibold uppercase before:w-2 before:h-2 before:bg-[#B130FF] text-[#B130FF]"
      >
        faza 4
      </span>
      <h4
        class="text-lg md:text-xl lg:text-2xl text-secondary font-bold text-secondary"
      >
        Wdrożenie
      </h4>
    </div>
    <p
      class="text-body text-sm md:text-base"
    >
      Po zatwierdzeniu projektu przechodzę do wdrożenia na serwer produkcyjny. Dodatkowo integruję narzędzia takie jak Google Analytics, Sentry, Google Search Console. Przeprowadzam również kompleksową optymalizację SEO.
    </p>
  </div>
</div>
`;
